apps/journeys-admin/pages/templates/[journeyId]/customize.tsx

```diff
@@ -25,14 +32,21 @@ function CustomizePage() {
   return (
     <>
       <NextSeo title={t('Customize Template')} />
-      <JourneyProvider
-        value={{
-          journey: data?.journey,
-          variant: 'default'
-        }}
+      <PageWrapper
+        user={user}
+        showMainHeader={false}
+        mainBodyPadding={false}
+        backgroundOverride="linear-gradient(to bottom, #1f2c430f, #2568994d)"
```

Comment:
instead of passing the styles in, we can just nest
```suggestion
        background="linear-gradient(to bottom, #1f2c430f, #2568994d)"
```

Reply:
Using the background prop of PageWrapper only sets the wrapper's stack backgroundColor, which is incompatible with linear-gradients. If we want a gradient color, we need to set the stack's background prop instead.